data/stats/get_flight_stats.py

```diff
@@ -0,0 +1,51 @@
+""" Gather flight statistics for each 
+    airport from given aiport flight data. 
+"""
+
+from dataclasses import dataclass, field
+from typing import TypedDict
+
+import pandas as pd
+
+
+class FlightDict(TypedDict):
+    aircraft_model: str
+    airline: str
+    destination_iata: str
+    destination_icao: str
+    flight_number: str
+    long_aircraft_name: str
+    origin_iata: str
+    origin_icao: str
+    scheduled_arrival: int
+    scheduled_departure: int
```

Comment:
I'm not sure I understand the purpose of these - how is this different to using the existing data types?